Makefile

```diff
@@ -35,7 +35,8 @@ controller: controller-bin controller-image
 importer: importer-bin importer-image
 push: push-importer push-controller
 test: functional-test unit-test
-functional-test: func-test-bin func-test-image func-test-run
+functional-test: func-test-run
```

Comment:
`make test` will now build and run the unit tests but only run the functional tests. Do we want this asymmetry?